### PR TITLE
Login and signup page

### PR DESCRIPTION
**Pull request by Naman-Saxena1**
Finished Login and Signup Pages

Login-Page
Signup-Page

### CODE REVIEW

File: Pages/Login-Page/login.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    max-width: fit-content;
+    align-self: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    color: var(--royal-blue);
+    padding: 5rem;
+    border-radius: 6px;
+    background-color: var();
```
Comment (rahulk31) [inline Pages/Login-Page/login.css]: bg color missing, may be that was unintentional..can be removed if so.

File: Pages/Signup-Page/signup.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    max-width: fit-content;
+    align-self: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    color: var(--royal-blue);
+    padding: 5rem;
+    border-radius: 6px;
+    background-color: var();
```
Comment (rahulk31) [inline Pages/Signup-Page/signup.css]: Here as well.

File: Pages/Signup-Page/signup.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                <label for="signup-input-name"><h4>Email address</h4></label>
+                <input id="signup-input-email" class="signup-form-input" type="email" placeholder="Email" required/>
+            </div>
+
+            <div class="signup-input-container">
+                <label for="signup-input-name"><h4>Password</h4></label>
+                <input id="signup-input-password" class="signup-form-input" type="password" placeholder="Password" required/>
+            </div>
+
+            <div class="remember-user-and-forgot-password-container">
```
Comment (rahulk31) [inline Pages/Signup-Page/signup.html]: I feel this classname is too extensive, can be improvised.

**Review by rahulk31 — COMMENTED** [review]
Overall looks great to me.

File: Pages/Login-Page/login.css (hunk 1 above)
Comment (Naman-Saxena1) [reply Pages/Login-Page/login.css]: Oh right, I forgot to remove it. Thanks

File: Pages/Signup-Page/signup.html (hunk 3 above)
Comment (Naman-Saxena1) [reply Pages/Signup-Page/signup.html]: Done, thanks for review 😄

**Review by rdthetha — COMMENTED** [review]
login and signup looks clean! Code also looks fine to me!

**Pull request merged**